docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -78,14 +108,14 @@ The easiest way to convert is to use Teams Toolkit.
 
     :::image type="content" source="../images/teams-toolkit-icon.png" alt-text="Teams Toolkit icon.":::
 
-1. Select **Create a new app**.
-1. In the **New Project** drop down, select **Outlook Add-in**.
+1. Select **Create a New App**.
+1. In the **New Project** drop down, select **Office Add-in**.
```

Comment:
can you also check for other instances of "drop down" that should say "dropdown menu"?

```suggestion
1. In the **New Project** dropdown menu, select **Office Add-in**.
```